src/test/codegen/remap_path_prefix/issue-73167-remap-std.rs

```diff
@@ -0,0 +1,16 @@
+// ignore-windows
+// ignore-tidy-linelength
+
+// compile-flags: -g  -C no-prepopulate-passes --remap-path-prefix=/home/USERNAME=REDACTED
+// aux-build:pseudo_std.rs
+
+extern crate pseudo_std;
+// Here we check that imported code from std has their path remapped.
+// If we were to use an actual std function to test, we'd have to remap {{cwd}}/library/...,
+// but that will break if rustc was built with remap-debuginfo = true in config.toml.
+// Hence we simulate the path rust-src would take in a real linux installation
+// by remapping the prefix inside pseudo_std.rs
+// CHECK: !DIFile(filename: "REDACTED/TOOLCHAIN/lib/rustlib/src/rust/library/std/src/pseudo_std.rs", directory: ""
```

Comment:
A `CHECK-NOT:` of the undesired prefixes across the entire file would be slightly more versatile than a check for a specific debuginfo metadata.